packages/containers/psammead-timestamp-container/src/index.jsx

```diff
@@ -36,7 +36,7 @@ const TimestampContainer = ({
       script={script}
     >
       {prefix ? `${prefix} ` : null}
-      {showRelativeTime(timestamp, isRelative, format, timezone)}
+      {showRelativeTime(timestamp, isRelative, format, timezone, locale)}
```

Comment:
Sorry, gunna be a huge pain, can you flip this please? Pass 'en' on line 34 to override the setting on line 29.

This way, we set to the correct locale, then pass the same locale again, defaulting to en. Ide rather we set the locale once (like on line 29), then only override if we want it to be different, like 'en'

Reply:
I might be going mad, but I'm not sure if I understand. Currently, `formatUnixTimestamp` defaults to `en` if the locale is not passed in which works for `datetime` value, but in that same function we need to be able to override the value of `locale` because of line 39 and 42 in `timestampUtilities`.

Reply:
Its ok, its ok as it is :)